### PR TITLE
SONAR-8940 support organizations in api/qualityprofiles/{add|remove}_project

### PR DESCRIPTION
**Pull request by simonbrandhof**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QProfileReference.java#L121](https://github.com/SonarSource/sonarqube/blob/d063cf8401b6d8ee224a87dfbc06ab2dd65a135f/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileReference.java#L121): Reduce the number of returns of this method 6, down to the maximum allowed 5. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1142)
1. ![MINOR][MINOR] [QProfileFactory.java#L148](https://github.com/SonarSource/sonarqube/blob/d063cf8401b6d8ee224a87dfbc06ab2dd65a135f/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L148): Remove this use of "selectByNameAndLanguage"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![INFO][INFO] [QProfileFactory.java#L133](https://github.com/SonarSource/sonarqube/blob/d063cf8401b6d8ee224a87dfbc06ab2dd65a135f/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L133): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L79](https://github.com/SonarSource/sonarqube/blob/d063cf8401b6d8ee224a87dfbc06ab2dd65a135f/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L79): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L98](https://github.com/SonarSource/sonarqube/blob/d063cf8401b6d8ee224a87dfbc06ab2dd65a135f/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L98): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QProfileReference.java#L122](https://github.com/SonarSource/sonarqube/blob/e50d1256107ded2af4cbef03313aa46bdc160280/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileReference.java#L122): Reduce the number of returns of this method 6, down to the maximum allowed 5. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1142)
1. ![MINOR][MINOR] [QProfileFactory.java#L166](https://github.com/SonarSource/sonarqube/blob/e50d1256107ded2af4cbef03313aa46bdc160280/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L166): Remove this use of "selectByNameAndLanguage"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![INFO][INFO] [QProfileFactory.java#L151](https://github.com/SonarSource/sonarqube/blob/e50d1256107ded2af4cbef03313aa46bdc160280/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L151): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L79](https://github.com/SonarSource/sonarqube/blob/e50d1256107ded2af4cbef03313aa46bdc160280/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L79): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L98](https://github.com/SonarSource/sonarqube/blob/e50d1256107ded2af4cbef03313aa46bdc160280/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L98): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QProfileReference.java#L122](https://github.com/SonarSource/sonarqube/blob/ae5a4b1eb60a0e756aa458f8a88c894a9748ca4c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileReference.java#L122): Reduce the number of returns of this method 6, down to the maximum allowed 5. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1142)
1. ![MINOR][MINOR] [QProfileFactory.java#L166](https://github.com/SonarSource/sonarqube/blob/ae5a4b1eb60a0e756aa458f8a88c894a9748ca4c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L166): Remove this use of "selectByNameAndLanguage"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![INFO][INFO] [QProfileFactory.java#L151](https://github.com/SonarSource/sonarqube/blob/ae5a4b1eb60a0e756aa458f8a88c894a9748ca4c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L151): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L79](https://github.com/SonarSource/sonarqube/blob/ae5a4b1eb60a0e756aa458f8a88c894a9748ca4c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L79): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L98](https://github.com/SonarSource/sonarqube/blob/ae5a4b1eb60a0e756aa458f8a88c894a9748ca4c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L98): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java [hunk 1]
```diff
@@ -146,10 +144,6 @@ private QualityProfileDto doCreate(DbSession dbSession, OrganizationDto organiza
 
   // ------------- DEFAULT PROFILE
 
-  public List<QualityProfileDto> getDefaults(DbSession session, OrganizationDto organization, Collection<String> languageKeys) {
-    return db.qualityProfileDao().selectDefaultProfiles(session, organization, languageKeys);
-  }
```
Comment (bartfastiel) [inline server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java]: 👍

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [QProfileReference.java#L122](https://github.com/SonarSource/sonarqube/blob/60ee3f65f20063801d62a6a3594e8af97206b03c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileReference.java#L122): Reduce the number of returns of this method 6, down to the maximum allowed 5. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1142)
1. ![MINOR][MINOR] [QProfileFactory.java#L166](https://github.com/SonarSource/sonarqube/blob/60ee3f65f20063801d62a6a3594e8af97206b03c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L166): Remove this use of "selectByNameAndLanguage"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![INFO][INFO] [QProfileFactory.java#L151](https://github.com/SonarSource/sonarqube/blob/60ee3f65f20063801d62a6a3594e8af97206b03c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java#L151): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L79](https://github.com/SonarSource/sonarqube/blob/60ee3f65f20063801d62a6a3594e8af97206b03c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L79): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)
1. ![INFO][INFO] [QProfileWsSupport.java#L98](https://github.com/SonarSource/sonarqube/blob/60ee3f65f20063801d62a6a3594e8af97206b03c/server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java#L98): Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**